container/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.9.12-slim-buster
+ARG VERSION
+
+WORKDIR /app
+
+COPY customrun.sh .
+
+COPY certificates/*.crt /usr/local/share/ca-certificates/
+RUN update-ca-certificates
+
+# set CERT paths
+#  - Requests Library
```

Comment:
Could you add details about these environment variables, why do we need to set them here? It may contain a link to the documentation as well